### PR TITLE
Read SPE PRESETS as metadata

### LINKED ISSUE #157: Remove SPE PRESETS warning
The following code:
```python
import becquerel as bq
s = bq.Spectrum.from_file('tests/samples/1110C NAA cave pottery.Spe')
```

Produces the warning
```
becquerel/parsers/spe_file.py:148: SpectrumFileParsingWarning: SpeFile has $PRESETS field, skipping 3 lines
  SpectrumFileParsingWarning)
```

This was introduced by #98 and its PR #135, however #81 and its PR #155 provide a more elegant approach: read these fields into the `metadata` dict.

### PR DESCRIPTION
**Pull request by markbandstra**
Remove the warning. Fix #157

### CODE REVIEW

**Comment by jccurtis** [comment]
Ah, Travis failed because the tests need to be updated to remove the warning requirement:

```python
=================================== FAILURES ===================================
___________________________ test_spe_presets_warning ___________________________
    def test_spe_presets_warning():
            with pytest.warns(
                    bq.parsers.SpectrumFileParsingWarning,
                    match='SpeFile has \$PRESETS field, skipping 3 lines'):
                bq.parsers.SpeFile(
>                   os.path.join(SAMPLES_PATH, 'digibase_5min_30_1.spe'))
tests/parsers_test.py:110: 
```

File: becquerel/parsers/spectrum_file.py [hunk 1]
```diff
@@ -85,8 +85,8 @@ def __str__(self):
         s += 'Realtime:              {:.2f} sec\n'.format(self.realtime)
         if len(self.metadata.keys()) > 0:
             s += 'Metadata:\n'
-            for key in self.metadata:
-                s += '    {} : {}\n'.format(key, self.metadata[key])
+            for key, value in self.metadata.items():
+                s += '    {} : {}\n'.format(key, str(value))
```
Comment (jccurtis) [inline becquerel/parsers/spectrum_file.py]:
```suggestion
                s += '    {} : {}\n'.format(key, value)
```

**Comment by jccurtis** [comment]
@markbandstra changes suggested and the tests need to be updated. Otherwise looks good! Thanks!

File: becquerel/parsers/spe_file.py [hunk 2]
```diff
@@ -142,21 +142,19 @@ def read(self, verbose=False):
...
-                    self.presets = []
-                    for _ in range(3):
-                        self.presets.append(lines[i])
-                        i += 1
                 elif lines[i].startswith('$'):
                     key = lines[i][1:-1]
                     i += 1
-                    if i < len(lines):
-                        value = lines[i]
+                    if key == 'PRESETS':
```
[the remark this answers is not in the document]
Comment (markbandstra) [reply becquerel/parsers/spe_file.py]: I'd prefer not to do this unless we change all of the other keywords, which are capitalized...

**Comment by markbandstra** [comment]
Oh man I always run all the tests, but forgot this one time.

File: becquerel/parsers/spe_file.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jccurtis) [reply becquerel/parsers/spe_file.py]: Okay sounds good!

**Comment by markbandstra** [comment]
Oops, I was caught not running all the tests again.

**Pull request merged**